recipes/tensorflow-quantum/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "tensorflow-quantum" %}
+{% set version = "0.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/quantum/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c6a4a72366334673c38cd06329f17889278db4dba8060beb891ef3c489fb72a8
+
+build:
+  number: 0
+  skip: true   # [ py<36 or win]
+  script:
+    - export TF_NEED_CUDA=0
+    - export TF_CUDA_VERSION=10.1
+    - ./configure.sh
+    - source gen-bazel-toolchain
+    - bazel build --crosstool_top=//bazel_toolchain:toolchain --cpu ${TARGET_CPU} release:build_pip_package
+    - bazel-bin/release/build_pip_package /tmp/tfquantum/
+    - ${PYTHON} -m pip install {{ name|lower }}-{{ version }}-py3-none-any.whl -vv --no-deps --no-index
+    - ./scripts/test_all.sh
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

`osx` may be fixed by
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
    - libprotobuf
```

---

Thanks, let's see.